Add to ros2 documentation one or multiple tutorials to create a first simulation with Webots (#1)
cyberbotics/ros2_documentation: source/Tutorials/Simulators/Tutorial-Create-Webots-Robot.md

![transform_up](./assets/tutorialCreateWebotsRobot/transform_up.png)

Add a `Shape` node to `children` of the new node.

Select `appearance` of the `Shape` node and add a `PBRAppearance` node.
Expand the `PBRAppearance` node, click on `baseColor` and chose a color for the body of your robot, so you will be able to distinguish this part.
Also set the roughness to 1 and metalness to 0 to have a better visual aspect.

Expand the `Shape` node, select `geometry` and add a `Cylinder` node.
In the `Cylinder` node set the height to `0.08` and the radius to `0.045`.
Click on the `Cylinder` node and in the `DEF` field (under the scene tree) type `BODY`.
This mechanism of Webots will permit you to reuse the parameters of this cylinder later.

You should have the following setup at this point:

![cylinder_param](./assets/tutorialCreateWebotsRobot/cylinder_param.png)

To the `boundingObject` field of your robot add a `Transform` node.
Set also the translation of this `Tranform` to `0 0 0.0415`.
Inside this new node add to the `children` field the `BODY` node you defined previously (you will find it in the `USE` field).

![cylinder_DEF](./assets/tutorialCreateWebotsRobot/cylinder_DEF.png)

> Add to your robot a `Physics` node to the corresponding field in order to enable the collision with other objects.

Reviewer: ```suggestion
Add to your robot a `Physics` node to the field `physics` in order to enable the collision with other objects.
```

Then select the field `controller` > `Select...` > `<extern>` to indicate to the simulator that the controller used will be extern to Webots (as you will use a ROS node).

You should end with this setup for the second part of the body of your robot:

![cylinder_bounding](./assets/tutorialCreateWebotsRobot/cylinder_bounding.png)

### Left wheel of the robot

Now you will add the wheels to your robot.
Let's start with the left one.
Select `Robot` > `children` and then add a `HingeJoint` node.

![hinge_joint](./assets/tutorialCreateWebotsRobot/hinge_joint.png)

> Then select `jointParameters` and add a `HingeJointParameters` node.

Reviewer: ```suggestion
Then, select `jointParameters` and add a `HingeJointParameters` node.
```

> Select `device`, add a `RotationalMotor` node and then configure the `HingeJointParameters` and `RotationalMotor` nodes like below.

Reviewer: ```suggestion
Select `device`, add a `RotationalMotor` node and configure the `HingeJointParameters` and `RotationalMotor` nodes like below.
```

![left_motor](./assets/tutorialCreateWebotsRobot/left_motor.png)

> `axis` will indicate around which axis the wheel should rotate and `anchor` indicates an offset.

Reviewer: ```suggestion
`axis` will indicate around which axis the wheel should rotate and `anchor` indicates an offset.
```
which offset?

> Next select `endPoint`, add a `Solid` node and set it up like here in a similar manner you did for the body of the robot:

Reviewer: ```suggestion
Select `endPoint`, add a `Solid` node and set it up like here in a similar manner you did for the body of the robot:
```

![left_wheel_solid](./assets/tutorialCreateWebotsRobot/left_wheel_solid.png)

> Then change the name to `left wheel` and add to the `boundingObject` a `WHEEL` node (you will find it again in the `USE` field).

Reviewer: ```suggestion
Change the name to `left wheel` and add to the `boundingObject` a `WHEEL` node (you will find it again in the `USE` field).
```

> Add to your robot a `Physics` node to its corresponding field once again in order to enable the collisions.

Reviewer: ```suggestion
Add to your `endPoint Solid` a `Physics` node to its field `physics` once again to enable the collisions.
```

You should end with the following for the `endPoint Solid` node.

![left_wheel_bounding](./assets/tutorialCreateWebotsRobot/left_wheel_bounding.png)

### Right wheel of the robot

> Creating the second wheel is bit easier as you can use the `DEF-USE` mechanism of Webots.

Reviewer: ```suggestion
Creating the second wheel is easier as you can use the `DEF-USE` mechanism of Webots.
```

> Select `Robot` > `children` > `HingeJoint` and then add a `HingeJoint` node.

Reviewer: ```suggestion
Select `Robot` > `children` > `HingeJoint` and add a `HingeJoint` node.
```

Once again select `jointParameters` and add a `HingeJointParameters` node.
Select `device`, add a `RotationalMotor` node and then configure the `HingeJointParameters` add `RotationalMotor` nodes like below.

![right_motor](./assets/tutorialCreateWebotsRobot/right_motor.png)

Select `endPoint`, add a `Solid` node and inside the new `children` add the `WHEEL` node.
Then change the name to `right wheel` and add to the `boundingObject` another `WHEEL` node.
Finally add to your robot a `Physics` node to its corresponding field.

![right_wheel](./assets/tutorialCreateWebotsRobot/right_wheel.png)

Congratulations!
You have now a simulation with a simple robot that can move around in the arena.
Do not forget to save your file with `Ctrl+S`.

![robot_body_wheel](./assets/tutorialCreateWebotsRobot/robot_body_wheel.png)

## Add sensors to your robot

Your robot is now able to move but it will not be able to avoid obstacle.

Let's add some sensors in order to make the robot able to detect front obstacles.
Start by adding a `DistanceSensor` node in the `children` field of your robot.

> Then configure the sensor as described in the following image in order to correctly place and model the sensor:

Reviewer: ```suggestion
Then, configure the sensor as described in the following image in order to correctly place and model the sensor:
```

![Sensor_1_1](./assets/tutorialCreateWebotsRobot/Sensor_1_1.png)

Next rename the sensor and configure the `lookupTable`, the `numberOfRays` and the `aperture` field.
You might need to press `lookupTable` > `Ctrl+Shift+A` in order to add a new line in the `lookupTable` field.

![Sensor_1_2](./assets/tutorialCreateWebotsRobot/Sensor_1_2.png)

> Creating the second sensor is straightforward and you can follow this structure:

Reviewer: ```suggestion
Creating the second sensor is straightforward and you can copy/paste the `ds0 DistanceSensor `node and follow this structure:
```
